src/stampy_nlp/settings.py

```diff
@@ -19,6 +19,12 @@
 RETRIEVER_MODEL_URL: str = os.getenv('RETRIEVER_MODEL_URL')
 LIT_SEARCH_MODEL_URL: str = os.getenv('LIT_SEARCH_MODEL_URL')
 
+ALLOWED_ORIGINS = os.getenv('ALLOWED_ORIGINS')
+if ALLOWED_ORIGINS:
+    ALLOWED_ORIGINS = [origin.strip() for origin in ALLOWED_ORIGINS.split(',')]
+else:
+    ALLOWED_ORIGINS = '*'
```

Comment:
use this as a decent default if nothing has been set. This means that CORS is effectively disabled by default